pages/api/docs/getAllRepos.ts

```diff
@@ -28,7 +31,18 @@ const getUserRepositoriesHandler = async (req: NextApiRequest, res: NextApiRespo
 	}
 	const eventProperties = { response_status: 200, result_length: repoList.length };
 	rudderStackEvents.track(session?.user?.id ?? "absent", "", 'user-repos-from-providers', { type: 'HTTP-200', eventStatusFlag: 1, eventProperties });    
-	return res.status(200).json({ repoList: repoList });
+  	res.status(200).json({ repoList: repoList });
+	const userId = getAuthUserId(session);
+	const userData: DbUser | null = await getUserById(userId).catch((err) => {
+		console.error('[getUserRepositoriesHandler] Error in getting user db data', err);
+		return null;
+	});
+	if (!userData?.topic_name) {
+		console.error(`[getUserRepositoriesHandler] Error in getting user topic from db`);
+		return;
+	}
```

Comment:
We don't need to add the topic name here. The same repos can also exist for other users and the install_id for them indicates that their Vibinex setup for these repositories are complete.

Reply:
done